kola/tests/ignition/resource.go

```diff
@@ -240,9 +238,6 @@ func init() {
 		Name:        "coreos.ignition.resource.s3.versioned",
 		Run:         resourceS3Versioned,
 		ClusterSize: 1,
-		// ESX: Currently Ignition does not support static IPs during the initramfs
-		// https://github.com/coreos/bugs/issues/2205 for DO
-		ExcludePlatforms: []string{"esx", "do"},
```

Comment:
The change above still excludes "do", this here not, maybe worth testing if it works now.

Reply:
That's normal, this test is flagged as `qemu` only:
> This does not need to run on all clouds because it is a special case of coreos.ignition.resource.remote

So the `ExcludePlatforms` was useless.

Reply:
Right, yet in case we want to run all tests on DO it would be good to know which ones are expected to work and which were explicitly excluded.